open-media-match/src/OpenMediaMatch/storage/interface.py

```diff
@@ -71,6 +72,26 @@ class ISignalTypeConfigStore(metaclass=abc.ABCMeta):
     def get_signal_type_configs(self) -> t.Mapping[str, SignalTypeConfig]:
         """Return all installed signal types."""
 
+    @abc.abstractmethod
+    def _create_or_update_signal_type_override(
+        self, signal_type: str, enabled_ratio: float
+    ) -> None:
+        """Create or update database entry for a signal type, setting a new value."""
+
+    @t.final
+    def create_or_update_signal_type_override(
+        self, signal_type: str, enabled_ratio: float
+    ) -> None:
+        """Update enabled ratio of an installed signal type."""
+        installed_signal_types = self.get_signal_type_configs()
+        if signal_type not in installed_signal_types:
+            raise ValueError(f"Unknown signal type {signal_type}")
+        if not (0.0 <= enabled_ratio <= 1.0):
```

Comment:
blocking: Move this to the database verifier (see the bank name validator for how to do this)